portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,109 @@
+let answer = -1;
+
+/**
+ * Gets the image and text from the position stored in index and 
+ * populates it with the fact stored in factIndex.  Populates a truth
+ * if isTruth is true and a lie otherwise
+ * 
+ *@param {number} index the position we will populate the fact
+ *@param {number} factIndex the position in the array of the 
+ * fact that will be populated
+ *@param {boolean} isTruth true if we will populate a truth; 
+ * false if we will populate a lie
+ */
+
+
+function loadInFact(index, factIndex, isTruth) {
+    let fact = isTruth ? truths[factIndex] : lies[factIndex];
+
+    switch(index) {
+        case 0:
+            document.getElementById('first-claim-text').innerHTML = fact[0];
+            document.getElementById('first-claim-image').src = fact[1];
+            break;
+        case 1:
+            document.getElementById('second-claim-text').innerHTML = fact[0];
+            document.getElementById('second-claim-image').src = fact[1];
+            break;
+        case 2:
+            document.getElementById('third-claim-text').innerHTML = fact[0];
+            document.getElementById('third-claim-image').src = fact[1];
+            break;
+    }
+}
+
+/**
+ * Populates the page with two true facts in randomized indexes
+ * 
+ *@return {Array} an array populated with two numbers containing
+ * locations of the true facts
+ */
+function loadInTruth() {
+
+    let first_truth_value = Math.floor(Math.random() * truths.length);
+    let second_truth_value = Math.floor(Math.random() * truths.length);
+    while (first_truth_value == second_truth_value) {
+        second_truth_value = Math.floor(Math.random() * truths.length);
+    }
+
+    let first_truth_index = Math.floor(Math.random() * 3);
+    let second_truth_index = Math.floor(Math.random() * 3);
+    while (first_truth_index == second_truth_index) {
+        second_truth_index = Math.floor(Math.random() * 3);
+    }
+
+    loadInFact(first_truth_index, first_truth_value, true);
+    loadInFact(second_truth_index, second_truth_value, true);
+
+    return [first_truth_index, second_truth_index];
+}
+
+/**
+ * Loads in a lie [used after loadInTruth]
+ * 
+ *@param {Array} truthIndices indices where true facts have 
+ * been populated
+ * @return {number} index populated by lie
+ */
+function loadInLie(truthIndices) {
+    //determine the location without a truth value
+    let lie_index = (3 - (truthIndices[0] + truthIndices[1]));
+    
+    let lie_value = Math.floor(Math.random() * lies.length);
+    loadInFact(lie_index, lie_value, false);
+
+    return lie_index;
+}
+
+/**
+ * Advances the game to the next question. Loads in the first question 
+ * game has not yet started. Otherwise, checks answer against user 
+ * selected anwer and alerts if a lie or truth has been chosen.
+ */
+function next()  {
+    let game_status = document.getElementById('game-status');
+    if (game_status.innerHTML=='Press Start to Begin!') {
+        game_status.innerHTML = 1;
+
```

Comment:
Nit: unnecessary whitespace, here and right below

Reply:
Changed